package-windows.bat

```diff
@@ -0,0 +1,20 @@
+@echo off
+rem calling this is required so cl.exe is on the path when native-image tries to compile our exe
+rem we can't do this from the travis bash script as the env variables set by this bat file are not
+rem propagated to the shell environment
+
+rem There are different potential paths for the vcvarsall.bat file (on the build machine it is apparently the "Enterprise" path)
+for %%f in (
+        "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvarsall.bat"
+        "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Auxiliary\Build\vcvarsall.bat"
+       ) do (
+         IF EXIST %%f (
+           call %%f x86_amd64
+           goto :vcvarsall_executed
+         )
+       )
+echo "Did not find Visual Studio vcvarsall.bat at one of the expected locations. Aborting now."
+exit
+:vcvarsall_executed
+
+./mvnw.cmd package
```

Comment:
What do we need the  `package` command for on Windows? `build-windows.bat` was sufficient for me until now.

Reply:
I had to run the package command before testing. At least that is what the documentation said (don't remember whether I tried without package):
> IntelliJ will not automatically recompile the native image after you made code changes. You must manually invoke ./mvnw package.

Reply:
I needed to do some research about the different Maven commands (for reference, I landed here: https://andresalmiray.com/maven-verify-or-clean-install/).

Apparently, `package` is the step before the integration tests. Previously, I would call `build-windows.bat` which uses `verify` which packages **and** runs the tests. This is somewhat redundant if I just want to package the exe and run the tests in IntelliJ.

So, it may be helpful to have this 🙂